lib/collection/request-body.js

```diff
@@ -214,9 +214,6 @@ _.assign(RequestBody.prototype, /** @lends RequestBody.prototype */ {
     toJSON () {
```

Comment:
Since we're not overriding anything in the `toJSON` function, can we remove the function override itself?